templates/field/field--node--oe-departments--oe-tender--full.html.twig

```diff
@@ -0,0 +1,12 @@
+{#
+/**
+ * @file
+ * Theme override for the Responsible department field template of the Call for tenders content type.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}
+  <span{{ item.attributes.addClass("ecl-u-type-color-blue-100") }}>{{ item.content }}</span>
```

Comment:
We don't need to color this if it's not a link, let's remove it.

Reply:
It was added because it appears in the design here https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=EWPP&title=Call+for+Tenders+Content+Type+Specification+-+Version+1.0.0